Extension/src/LanguageServer/cppBuildTaskProvider.ts

```diff
@@ -55,6 +55,12 @@ export class CppBuildTaskProvider implements TaskProvider {
         return undefined;
     }
 
+    public resolveInsiderTask(_task: CppBuildTask): CppBuildTask | undefined {
```

Comment:
Should the "resolveTask" method be removed? It doesn't appear to be used anymore.

Reply:
Ok

Reply:
TypeScript doesn't show any indication that the method is overriding a base class method (unlike C++'s override/virtual), so I didn't notice it was an override.